client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -533,69 +589,36 @@ const pollForLabelsPurchase = ( orderId, siteId, dispatch, getState, labels ) =>
 	} ) );
 	const state = getShippingLabel( getState(), orderId, siteId );
 	const printUrl = getPrintURL( state.paperSize, labelsToPrint );
+	const showSuccessNotice = () => {
+		dispatch( NoticeActions.successNotice( translate(
+			'Your %(count)d shipping label was purchased successfully',
```

Comment:
Since we're making changes that effect these lines, what do you think about changing the singular case to exclude the number of labels?

E.g. `Your shipping label was purchased successfully` vs `Your 1 shipping label was purchased successfully`

Reply:
Sounds like an opportunity to improve the linter. 🙂 

IMO, removing the number is the better UX. If we agree on that we can use comments to have the linter ignore this line until we fix the linter.

_(cc: @kellychoffman @jameskoster for designer opinion)_

Reply:
> Your shipping label was purchased successfully. 

I like this. Also, friendly PSA/reminder to always add screenshots to PRs that have UI changes.